fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java

```diff
@@ -1086,7 +1086,7 @@ public void setEnableParallelMerge(boolean enableParallelMerge) {
     private String queryExcludingMVNames = "";
 
     @VarAttr(name = QUERY_INCLUDING_MV_NAMES, flag = VariableMgr.INVISIBLE)
-    private String queryincludingMVNames = "";
+    private String queryIncludingMVNames = "";
```

Comment:
use includingMVNames directly？
query prefix is useless?

Reply:
`queryIncludingMVNames ` indicates this query can use which mvs to rewrite.
`queryExcludingMVNames ` indicates this query can not use which mvs to rewrite.